examples/ex1.cpp

```diff
@@ -79,6 +81,8 @@ int main(int argc, char *argv[])
                   "--no-static-condensation", "Enable static condensation.");
    args.AddOption(&pa, "-pa", "--partial-assembly", "-no-pa",
                   "--no-partial-assembly", "Enable Partial Assembly.");
+   args.AddOption(&ea, "-ea", "--element-assembly", "-no-ea",
```

Comment:
Here (and in ex1p, ex9, ex9p) I think it makes more sense to do a `char *` optioin, maybe `"-al", "--assembly-level"` with choices `element`, `partial`, `full`, etc. instead of having a bunch of bool options.

Reply:
I did your suggested change, tell me if you like it.
@tzanio @v-dobrev Can you check that you like this new way to set the assembly level for `ex1`, `ex1p`, `ex9` and `ex9p`. (This could potentially be propagated to other examples)

Reply:
I like this way better, but you're right, it may affect other examples so it's best to be consistent.